Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3
+
+ENV PYTHONUNBUFFERED 1
+
+RUN apt-get update -y
+RUN apt-get -y install binutils libproj-dev gdal-bin
+
+RUN mkdir /ais
+WORKDIR /ais/
+
+ADD requirements.txt /ais/
+RUN pip install -r requirements.txt
+
+ADD . /ais/
```

Comment:
Revert the commit of this file as Docker is depended on PR #673, make sure to now commit this file.

Reply:
Remove it from the PR, but you can have the change locally.